lib/index.js

```diff
@@ -46,7 +46,7 @@ module.exports = function (ops, coverage) {
       bin = join(require.resolve('istanbul'), '..', istPkg.bin.istanbul);
     }
     // Execute Mocha, stdin and stdout are inherited
-    this._child = proc.fork(bin, args.concat(this._files), {env:env});
+    this._child = proc.fork(bin, args.concat(this._files), {env:env, cwd: ops.cwd });
```

Comment:
Any chance you can treat this like the other non-mocha options? Save it off to a variable up above and then add `cwd` to the omit call.